tests/utils/test_cdb_utils.py

```diff
@@ -0,0 +1,42 @@
+import unittest
+import numpy as np
+from tests.helper import ForCDBMerging
+from medcat.utils.cdb_utils import merge_cdb
+
+
+class CDBMergeTests(unittest.TestCase):
+
+    def setUp(self) -> None:
```

Comment:
Sorry for the confusion, but I think it would make more sense to have this be the static setup.
That way we'd only be setting up things once instead of before each test (which saves time on test execution).

While setting up things before every test would make sense if we were making changes to the stuff we set up, we're not doing that here.
So I'd be inclined to use
```
@classmethod
def setUpClass(cls):
```